theme_bewise/static/src/js/tour.js

```diff
@@ -48,8 +42,4 @@ wTourUtils.registerThemeHomepageTour("bewise_tour", [
     wTourUtils.dragNDrop(snippets[4], 'top'),
     wTourUtils.dragNDrop(snippets[5], 'top'),
     wTourUtils.dragNDrop(snippets[6], 'top'),
-    wTourUtils.clickOnSnippet(snippets[6], 'top'),
-    wTourUtils.changeOption('BackgroundShape', 'we-toggler', _t('Background Shape')),
-    wTourUtils.selectNested('we-select-page', 'BackgroundShape', ':not(.o_we_pager_controls)', _t('Background Shape')),
```

Comment:
Indeed. I also think we should keep this stuff. 
I guess it's because the snippet doesn't have a shape anymore.
So you can move it to another snippet (e.g. s_quotes_carousel).

Reply:
Agreed :)

Reply:
I've added it on another snippet 🙂